### DISCUSSION TITLE
Date format

### DISCUSSION THREAD

**Discussion by GilOliveira** [joaomlourenco/novathesis#282] [Q&A] [answered]
Hello everyone!

I'm sorry if this has already been addressed (couldn't find it here or on the Facebook group).
How do I change the date format of the "visited on" section of the bibliography? It's defaulting to the MM/DD/YYYY American-style format, which can be confusing to European readers. Preferably I would like it to be [ISO 8601](https://www.w3.org/QA/Tips/iso-date) YYYY-MM-DD to avoid ambiguity. Thanks!

![image](https://user-images.githubusercontent.com/1360438/151552256-3c5ca1c9-2b75-4679-8335-6ba46e69ff8a.png)

_PS: Maybe this could be the default behaviour in future versions..._

**Comment by joaomlourenco** [comment, marked as answer]
@GilOliveira 
I don't remember this issue being discussed.  But thank you for _searching first and asking afterwards_!  :) 

**I did not try this solution… but check if it works and let us know!**  If it does, please remember to make this answer as the right answer.

Try editing `Config/2_biblatex.tex` and add
```latex
\ntbibsetup{urldate=iso,date=iso}
```
If the `iso` is not recognized (depends on the Biblatex version), replace it with `edtf`).

**Comment by GilOliveira** [comment]
Works perfectly! Thank you so much Professor @joaomlourenco!
